src/models/appconfig.ts

```diff
@@ -39,6 +39,17 @@ export enum NewVersionCode {
   Downloaded = 4,
 }
 
+export type KubePermissions = {
+  resourceName: string;
```

Comment:
is the resource name enough here? or should it be `kind` + `apiVersion` + `name` ?

Reply:
from that I tested in rbac you could set specific permissions per resource name